settings.py

```diff
@@ -160,6 +160,7 @@
     'auditcare.middleware.AuditMiddleware',
     'no_exceptions.middleware.NoExceptionsMiddleware',
     'corehq.apps.locations.middleware.LocationAccessMiddleware',
+    'two_factor.middleware.threadlocals.ThreadLocals',
```

Comment:
I think that this is technically only needed for the twilio gateway, but just went ahead and added in here.

http://django-two-factor-auth.readthedocs.io/en/stable/configuration.html#twilio-gateway